### PR TITLE
Test for Add function and delele function

### PR DESCRIPTION
**Pull request by Alejandro-Bernal-M**
# Tests with jest

We implemented the tests for the add function and for the remove function.

https://github.com/Alejandro-Bernal-M/To-do-list/blob/ddf7c21bd5a47e3d89fbd4bdb76b8017f805a8c4/src/addAndDelete.test.js#L4-L18

image

### CODE REVIEW

File: src/addAndDelete.test.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  });
+});
+
+describe('Remove test', () => {
+  test('should remove from localstorage', () => {
+    localStorage.setItem('tasks', JSON.stringify([{ completed: false, description: 'task1', index: 1 }]));
+    remove('task1');
+    expect(JSON.parse(localStorage.getItem('tasks'))).toEqual([]);
+  });
+});
```
Comment (Gambit142) [inline src/addAndDelete.test.js]: - [x] Great work testing your add and remove functions. When I run the `npm run test` command, I get an error that `Jest-environment-dom` can not be found. You need to include `"jest-environment-jsdom": "^29.4.3"` in your `package.json` file or advise those who clone your repository to run `npm install -D jest-environment-jsdom` after running `npm install` in your _**README.md**_ file. 👍 😎 

image

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @Alejandro-Bernal-M @getssh 👋, 

Good job so far!

## Highlights
- All linters checks passed ✔️ 
- Test function for adding tasks ✔️ 
- Tests function for removing tasks ✔️ 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please, remember to tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/addAndDelete.test.js (hunk 1 above)
Comment (Alejandro-Bernal-M) [reply src/addAndDelete.test.js]: Hi @Gambit142 , Ok! we added it to the package.json. Thank you for the review, now we are going to request another one. 

image

**Review by binhussen — APPROVED** [review]
Hi @Alejandro-Bernal-M , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used Git flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Alejandro-Bernal-M** [comment]
Thank you

**Pull request merged**